docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -229,24 +261,43 @@ spec:
 For details, see the [LogPipeline specification file](https://github.com/kyma-project/kyma/blob/main/components/telemetry-operator/apis/telemetry/v1alpha1/logpipeline_types.go).
 
 | Parameter | Type | Description |
-|-|-|-|
-| filters | []object | A list of [Fluent Bit filters](https://docs.fluentbit.io/manual/pipeline/filters) to apply to the logs processed by the pipeline. Filters are executed in sequence, as defined. They are executed before logs are buffered, and with that, are not executed on retries.|
-| filters[].custom | string | The actual filter definition in the syntax of Fluent Bit.|
-| output | object | [Fluent Bit outputs](https://docs.fluentbit.io/manual/pipeline/outputs).|
-| output.custom | string | The actual output definition in the syntax of Fluent Bit.|
+|---|---|---|
+| filters | []object | List of [Fluent Bit filters](https://docs.fluentbit.io/manual/pipeline/filters) to apply to the logs processed by the pipeline. Filters are executed in sequence, as defined. They are executed before logs are buffered, and with that, are not executed on retries.|
+| filters[].custom | string | Actual filter definition in the Fluent Bit syntax.|
+| output | object | [Fluent Bit output](https://docs.fluentbit.io/manual/pipeline/outputs) where the logs should be pushed to. Only one output can be specified. |
+| output.http | object | [Fluent Bit http output](https://docs.fluentbit.io/manual/pipeline/outputs/http). |
+| output.http.compress | string | Payload compression mechanism. |
+| output.http.dedot | boolean | Indicates if *dedotting* (replacing dots with underscores) should be applied on the log field names. |
```

Comment:
I think we apply that only on kubernetes.labels and kubernetes.annotations. Please double-check

Reply:
yes, will adjust it 👍